tv-shows-app/src/components/shared/PasswordInput/PasswordInput.tsx

```diff
@@ -19,14 +18,8 @@ export default function PasswordInput({ name, register }: IPasswordInputProps) {
   return (
     <InputGroup>
       <Input
-        data-testid={name}
-        {...register(name)}
+        {...rest}
```

Comment:
Can I set {...register(name)} directly in the parent component or do I use {...rest} only for passing placeholder value in this case?

Reply:
You can use the same approach for passing register.
There will be an issue I think which I forgot to mention, and that is something like `Function components can't accept ref values`.

To avoid that, you need can use forwardRef like in [these docs](https://v2.chakra-ui.com/community/recipes/as-prop#option-1-using-forwardref-from-chakra-uireact).

You can try it out, but I'll explain it to everyone in the next lecture.